<?xml version="1.0" encoding="UTF-8"?>
<launch>
    <arg name="chessboardCornersCountX" default="9"/>
    <arg name="chessboardCornersCountY" default="6"/>
    <arg name="sizeOfSquareSide" default="24"/>
    <arg name="frameWidth" default="640"/>
    <arg name="frameHeight" default="480"/>
    <param name="camera_calibrated" command="$(find camera_calibration)/scripts/calibrate_camera.py
    --chessboardCornersCountX $(arg chessboardCornersCountX)
    --chessboardCornersCountY $(arg chessboardCornersCountY)
    --sizeOfSquareSide $(arg sizeOfSquareSide)
    --frameWidth $(arg frameWidth)
    --frameHeight $(arg frameHeight)"/>
</launch>
